src/world.rs

```diff
@@ -297,6 +297,12 @@ impl Generation {
         Generation(v)
     }
 
+    /// Returns the id of the generation.
+    #[inline]
+    pub fn get_id(&self) -> i32 {
```

Comment:
Oh, I'm sorry, I couldn't read the code properly first because I was using my phone. So the correct getter would be `.id()`. Additionally, the `Entity` getters should be renamed.